src/app/(affiliate)/affiliate/events/page.tsx

```
@@ -36,175 +37,99 @@
-      description: 'Intimate chamber music performances in a cozy setting.',
-    },
-  ],
-  categories: [
-    { name: 'Classical', events: 4, revenue: 21350, percentage: 47.2 },
-    { name: 'Holiday', events: 2, revenue: 7200, percentage: 15.9 },
-    { name: 'Gala', events: 2, revenue: 6100, percentage: 13.5 },
-    { name: 'Festival', events: 2, revenue: 4800, percentage: 10.6 },
-    { name: 'Chamber', events: 2, revenue: 3200, percentage: 7.1 },
-  ],
+type eventByStatusItem = {
+  upcoming: Number
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)